addons/sale_pdf_quote_builder/models/ir_actions_report.py

```diff
@@ -138,8 +138,9 @@ def _get_form_fields_mapping(self, order):
         }
 
         # Adding fields from each line, prefixed by the line_id to avoid conflicts
-        for line in order.order_line:
```

Comment:
I mean: `for line in order.order_line.filtered(...):` =]
List comprehension here makes no sense, we don't need it.

Reply:
Sure, I had only used list comprehension there initially because I was under the impression it was faster than filter

Reply:
When you need to assign: list comprehension is slightly better, but otherwise, you can use filter.
But keep in mind that the code also needs to be clear and comprehensible. This whole flow isn't the most obvious one, so it's better to not uselessly over-complicate it even more.